### ISSUE TITLE
Could not find a version that satisfies the requirement ai-edge-litert

### ISSUE THREAD

**Issue by fracico-tech** [ultralytics/ultralytics#19805]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Export

### Bug

```
from ultralytics import YOLO

# Load the YOLO11 model
model = YOLO("yolo11n.pt")

# Export the model to TFLite format
model.export(format="tflite")
```

Just by running this code I get:
```
ERROR: Could not find a version that satisfies the requirement ai-edge-litert>=1.2.0 (from versions: none)
ERROR: No matching distribution found for ai-edge-litert>=1.2.0
```

### Environment

```
Ultralytics 8.3.94 ðŸš€ Python-3.10.11 torch-2.6.0+cpu CPU (AMD Ryzen 9 5950X 16-Core Processor)
Setup complete âœ… (32 CPUs, 63.9 GB RAM, 308.9/1861.7 GB disk)

OS                  Windows-10-10.0.26100-SP0
Environment         Windows
Python              3.10.11
Install             pip
Path                C:\Users\ciko9\AppData\Local\Programs\Python\Python310\Lib\site-packages\ultralytics
RAM                 63.91 GB
Disk                308.9/1861.7 GB
CPU                 AMD Ryzen 9 5950X 16-Core Processor
CPU count           32
GPU                 None
GPU count           None
CUDA                None

numpy               âœ… 2.1.1<=2.1.1,>=1.23.0
matplotlib          âœ… 3.10.1>=3.3.0
opencv-python       âœ… 4.11.0.86>=4.6.0
pillow              âœ… 11.1.0>=7.1.2
pyyaml              âœ… 6.0.2>=5.3.1
requests            âœ… 2.32.3>=2.23.0
scipy               âœ… 1.15.2>=1.4.1
torch               âœ… 2.6.0>=1.8.0
torch               âœ… 2.6.0!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision         âœ… 0.21.0>=0.9.0
tqdm                âœ… 4.67.1>=4.64.0
psutil              âœ… 7.0.0
py-cpuinfo          âœ… 9.0.0
pandas              âœ… 2.2.3>=1.1.4
seaborn             âœ… 0.13.2>=0.11.0
ultralytics-thop    âœ… 2.0.14>=2.0.0
```

### Minimal Reproducible Example

Already shared above

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @fracico-tech, thank you for bringing this to our attention 🚀! We recommend checking out the [Ultralytics Docs](https://docs.ultralytics.com/) for detailed guidance on tasks like exporting models. You can find export-specific details in the [Export Documentation](https://docs.ultralytics.com/modes/export/), which might address your issue.

If this is a 🐛 Bug Report, thank you for including your environment details and code snippet 🙌. This is helpful! To further assist us, please ensure you've followed these steps:
1. Upgrade to the latest version of the `ultralytics` package to see if the issue persists:
```bash
pip install -U ultralytics
```
2. If the issue remains, confirm that your Python environment meets all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml), including compatibility with [PyTorch](https://pytorch.org/get-started/locally/) and other dependencies.

It seems the error relates to `ai-edge-litert`. This might indicate a missing or incompatible dependency. Please ensure your environment is configured correctly and meets the necessary installation criteria.

For more context, you can run YOLO in various supported environments:
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **AWS** Deep Learning AMI: See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

If this is a ❓ Question, please provide additional details if needed, such as full logs or a more detailed environment setup. This information will help clarify the issue.

Finally, feel free to join our community for additional support:
- [Discord](https://discord.com/invite/ultralytics) 🎧 for real-time discussions
- [Discourse](https://community.ultralytics.com/) for in-depth conversations
- [Subreddit](https://reddit.com/r/Ultralytics) to share insights and experiences with other users

This is an automated response, but rest assured, an Ultralytics engineer will review and assist you further as soon as possible. Thank you for your patience and for using Ultralytics! 🚀

**Comment by glenn-jocher** [comment]
@fracico-tech this package is required in the latest versions for TF export. You can downgrade to an earlier version of `ultralytics` for legacy TF export without this package, and I'd recommend requesting Windows support with the package author (Google) at https://github.com/google-ai-edge/LiteRT

**Comment by JinYu1998** [comment]
> [@fracico-tech](https://github.com/fracico-tech) this package is required in the latest versions for TF export. You can downgrade to an earlier version of `ultralytics` for legacy TF export without this package, and I'd recommend requesting Windows support with the package author (Google) at https://github.com/google-ai-edge/LiteRT此软件包是最新版本 TF 导出所必需的。您可以降级到早期版本的`ultralytics`以进行旧版 TF 导出，而无需使用此软件包，我建议您在https://github.com/google-ai-edge/LiteRT上向软件包作者 (Google) 请求 Windows 支持

I use `yolov8_int.tflite` from https://github.com/ultralytics/yolo-flutter-app/tree/main/example/assets. It runs at 60+ ms in my android app.
But if I use the following code. 

```
from ultralytics import YOLO

model = YOLO('yolov8n.pt')

model.export(format='tflite', imgsz=640, int8=True)
```

Then it runs at 200ms in the app on Android . 
Why is this, I am using the latest version of `ultralytics`.

**Comment by JinYu1998** [comment]
> > [@fracico-tech](https://github.com/fracico-tech) this package is required in the latest versions for TF export. You can downgrade to an earlier version of `ultralytics` for legacy TF export without this package, and I'd recommend requesting Windows support with the package author (Google) at [https://github.com/google-ai-edge/LiteRT此软件包是最新版本](https://github.com/google-ai-edge/LiteRT%E6%AD%A4%E8%BD%AF%E4%BB%B6%E5%8C%85%E6%98%AF%E6%9C%80%E6%96%B0%E7%89%88%E6%9C%AC) TF 导出所必需的。您可以降级到早期版本的`ultralytics`以进行旧版 TF 导出，而无需使用此软件包，我建议您在[https://github.com/google-ai-edge/LiteRT上向软件包作者](https://github.com/google-ai-edge/LiteRT%E4%B8%8A%E5%90%91%E8%BD%AF%E4%BB%B6%E5%8C%85%E4%BD%9C%E8%80%85) (Google) 请求 Windows 支持
> 
> I use `yolov8_int.tflite` from https://github.com/ultralytics/yolo-flutter-app/tree/main/example/assets. It runs at 60+ ms in my android app. But if I use the following code.
> 
> ```
> from ultralytics import YOLO
> 
> model = YOLO('yolov8n.pt')
> 
> model.export(format='tflite', imgsz=640, int8=True)
> ```
> 
> Then it runs at 200ms in the app on Android . Why is this, I am using the latest version of `ultralytics`.

BTW, is there any old version that can export tflite models stably?

**Comment by glenn-jocher** [comment]
Great catch on identifying the input size discrepancy! The `imgsz` parameter significantly impacts both accuracy and inference speed, as it directly affects the model's computational complexity. For optimal performance, ensure your training and export image sizes match your deployment requirements. If others encounter similar issues, we recommend verifying all export parameters against the TFLite export documentation. 😊

**Comment by jakubrymuza** [comment]
It seems to me like this library is not available on windows (https://pypi.org/project/ai-edge-litert/#files only has files for linux and macos). I got the same bug on windows, but it worked correctly on linux (Debian).

**Comment by glenn-jocher** [comment]
@jakubrymuza exactly right - `ai-edge-litert` currently only supports Linux and macOS platforms. The package availability you linked confirms this Windows limitation. For Windows users needing TFLite export, the current workaround is downgrading to an earlier Ultralytics version or using a Linux environment for the export process.

**Comment by geneconceja** [comment]
> [@jakubrymuza](https://github.com/jakubrymuza) exactly right - `ai-edge-litert` currently only supports Linux and macOS platforms. The package availability you linked confirms this Windows limitation. For Windows users needing TFLite export, the current workaround is downgrading to an earlier Ultralytics version or using a Linux environment for the export process.

what specific earlier version?

**Comment by glenn-jocher** [comment]
Use ultralytics==8.2.103 (any 8.2.x works) — these versions use the legacy TFLite export path and don’t require ai-edge-litert on Windows; install with: pip install ultralytics==8.2.103.

**Comment by hacktronics** [comment]
the previous model ultralytics==8.2.103 does not seems to load yolov11 model, which i need to convert to tfjs. Tried converting on google collab, and the conversion is success, but yolo can't seems to load the web model folder, not a path issue.

**Comment by glenn-jocher** [comment]
YOLO11 isn’t supported in 8.2.x, and TF.js exports aren’t loadable with the Ultralytics Python/CLI—use a current Ultralytics release on Linux/Colab to export, then load the generated model.json with TensorFlow.js in your web/Node app; for Python inference use SavedModel/PB/TFLite instead; details are in the TF.js section of the Ultralytics Integrations page: https://docs.ultralytics.com/integrations/
